src/flameGraph.cpp

```diff
@@ -484,6 +636,73 @@ double FlameGraph::printFrame(std::ostream& out, const std::string& name, const
     return framewidth;
 }
 
+void FlameGraph::printTreeHeader(std::ostream& out, long total) {
+    char buf[sizeof(TREE_HEADER) + 256];
+    std::string treeName = "Calltree";
+    if(_reverse){
+        treeName = "Backtrace";
+    } 
+    sprintf(buf, TREE_HEADER, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse,_reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, treeName.c_str(), total, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse, _reverse);     
```

Comment:
This looks odd :)
Are all these `_reverse` parameters still needed? Even if you plan to merge multiple tree views later, I'm sure JS functions can be tuned to work only with elements from the right context.